spec/mailers/previews/claims_api_unsuccessful_report_mailer_preview.rb

```diff
@@ -70,40 +70,44 @@ def call_factories
   end
 
   def make_claims
-    FactoryBot.build(:auto_established_claim_v2, :errored)
-    FactoryBot.build(:auto_established_claim, :errored)
+    # ClaimsApi::AutoEstablishedClaim.where(created_at: @from..@to).destroy_all
+    FactoryBot.create(:auto_established_claim_v2, :errored)
```

Comment:
With build you can't query the database as we do to get the totals for the report.